### PR TITLE
Add more response edits

### PR DESCRIPTION
**Pull request by hollenberry**
This PR continues the response adjustments started in #8 , and brings over the comments for @JasonEtco 's attention noted in #7 .

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^title: *\w(?![ \t]*#).*$/gm'
+    left: '/^title: *\w(?![ \t]*#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  repo: *lab-starter-template(?!#).*$/gm'
+    left: '/^  repo: *lab-starter-template(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 3]
```diff
@@ -108,7 +108,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  name: *\w(?![ \t]*#).*$/gm'
+    left: '/^  name: *\w(?![ \t]*#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 4]
```diff
@@ -162,7 +162,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^- type: *createIssue(?!#).*$\n  title: *(?![ \t]*#).*$\n. body: *welcome-text.md(?!#).*$/gm'
+    left: '/^- type: *createIssue(?!#).*$\n  title: *(?![ \t]*#).*$\n. body: *welcome-text.md(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 5]
```diff
@@ -191,7 +191,7 @@ steps:
     action_id: fileContents
     filename: 'responses/welcome-text.md'
   - type: gate
-    left: '/\w+/gm'
+    left: '/\w+/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 6]
```diff
@@ -233,7 +233,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  description:(?![ \t]*#).*$/gm'
+    left: '/^  description:(?![ \t]*#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 7]
```diff
@@ -263,7 +263,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  event: *pull_request.opened(?!#).*$/gm'
+    left: '/^  event: *pull_request.opened(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 8]
```diff
@@ -293,7 +293,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  - type: *gate(?!#).*$/gm'
+    left: '/^  - type: *gate(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 9]
```diff
@@ -347,7 +347,7 @@ steps:
   #   action_id: fileContents
   #   filename: 'config.yml'
   # - type: gate
-  #   left: '/^    left: *'%payload.pull_request.title%'(?!#).*$\n    operator: *===(?!#).*$\n.   right: *Add name to README(?!#).*$/gm'
+  #   left: '/^    left: *'%payload.pull_request.title%'(?!#).*$\n    operator: *===(?!#).*$\n.   right: *Add name to README(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 10]
```diff
@@ -389,7 +389,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^  - type: *respond(?!#).*$\n    with: *pr-opened.md(?!#).*$/gm'
+    left: '/^  - type: *respond(?!#).*$\n    with: *pr-opened.md(?!#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml [hunk 11]
```diff
@@ -431,7 +431,7 @@ steps:
     action_id: fileContents
     filename: 'config.yml'
   - type: gate
-    left: '/^tagline:(?![ \t]*#).*$\ndescription:(?![ \t]*#).*$/gm'
+    left: '/^tagline:(?![ \t]*#).*$\ndescription:(?![ \t]*#).*$/gm'
```
Comment (hollenberry) [inline config.yml]: ^

File: config.yml (hunk 3 above)
Comment (JasonEtco) [reply config.yml]:
```suggestion
    left: '/^\s{2}name:\s*\w(?![\s\t]+#).*$/gm' 
```

**Comment by hollenberry** [comment]
> Thanks, @hollenberry! What are your thoughts on 🚢ing this, merging master into #10, and continuing the work there?

Sounds good @hectorsector . Merging to keep it tidy, and since #10 is closed, I'll open a new PR with today's changes.

**Pull request merged**